cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
The Debug Toolbar is included in the URL patterns. Ensure that this is only enabled in a development environment and not in production, as it can expose sensitive information.